clients/src/main/resources/common/message/AssignReplicasToDirsRequest.json

```diff
@@ -0,0 +1,40 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 73,
+  "type": "request",
+  "listeners": ["controller"],
+  "name": "AssignReplicasToDirsRequest",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    { "name": "BrokerId", "type": "int32", "versions": "0+", "entityType": "brokerId",
+      "about": "The ID of the requesting broker" },
+    { "name": "BrokerEpoch", "type": "int64", "versions": "0+", "default": "-1",
+      "about": "The epoch of the requesting broker" },
+    { "name": "Directories", "type":  "[]DirectoryData", "versions": "0+", "fields":  [
+      { "name":  "Id", "type": "uuid", "versions": "0+", "about": "The ID of the directory" },
+      { "name": "Topics", "type": "[]TopicData", "versions": "0+", "fields": [
+        { "name":  "Name", "type": "string", "versions": "0+", "entityType": "topicName",
```

Comment:
Agreed, both here and in the response json.  The KIP will also need an update since it indicates name in both places.  We must have missed that on the discussion/vote threads.

Reply:
Correct, changed. I've also updated the KIP.